scripts/tests/test-ci-all.sh

```diff
@@ -183,6 +189,7 @@ tests_to_run_in_parallel=(
   "devimint_cli_test_single"
   "load_test_tool_test"
   "recoverytool_tests"
+  "guardian_backup"
```

Comment:
@bradleystachurski /@dpc am I holding it correctly?

Reply:
Yep, this is perfect. Do you know if this test takes a while? If so, we might want to move it higher up so `parallel` starts it sooner.

Reply:
Nah, running it locally was under 30s I'd say